package.json

```diff
@@ -278,7 +270,13 @@
                     "name": ""
                 }
               ]
-        }
+        },
+        "viewsWelcome": [
+            {
+              "view": "lf-lang-projects",
+              "contents": "No Lingua Franca project found. [Learn more](https://www.lf-lang.org/docs/) about setting up a Lingua Franca project structure.\n[Open Lingua Franca Project](command:linguafranca.openFolder)"
```

Comment:
Should we point to a specific page?

Reply:
Yes, that’s the idea. Do you know which specific page?

Reply:
https://www.lf-lang.org/docs/tools/code-extension/#usage but this page it rather incomplete. Also, there is no functionality to create a new project, or a description of how to do so. Should be easy to let `lingo` do it via VS Code once it's available as a library...